src/main/resources/templates/fragments.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      xmlns:sec="https://www.thymeleaf.org/thymeleaf-extras-springsecurity5" lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Fragments</title>
+</head>
+<body>
+<div th:fragment="header">
+    <nav class="navbar navbar-expand-lg navbar-light" style="background: lightblue">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#">Panel eksperta</a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                <div class="navbar-nav ">
+                    <a class="nav-link active" aria-current="page" th:href="|/|">Obstaw</a>
+                    <a class="nav-link active" aria-current="page" th:href="|/rules|">Zasady</a>
+                    <a sec:authorize="isAuthenticated()" class="nav-link active" aria-current="page" th:href="|/results|">Wyniki</a>
+                    <a sec:authorize="isAuthenticated()" class="nav-link active" aria-current="page" th:href="|/profile|">Edytuj profil</a>
+                    <a sec:authorize="isAuthenticated()" class="nav-link active" aria-current="page" th:href="|/logout|">Wyloguj</a>
+                    <a sec:authorize="isAuthenticated() == false" class="nav-link active" aria-current="page" th:href="|/register|">Zarejestruj</a>
+                    <a sec:authorize="isAuthenticated() == false" class="nav-link active" aria-current="page" th:href="|/login|">Zaloguj</a>
+                </div>
+            </div>
+        </div>
+    </nav>
+</div>
+<div style="background: burlywood; margin-top: 30px" th:fragment="footer">
+    <nav class="navbar navbar-expand-lg navbar-light">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#">Panel administracyjny</a>
```

Comment:
Dziwnie się to zachowuje :)
Na większym ekranie jest to link, po kliknięciu którego nic się nie dzieje, a na mobilnym można kliknąć ikonkę hamburgera, ale wtedy rozwija się główne menu, a nie menu admina.
Nie ucinam punktów, ale warto jeszcze chyba na to rzucić okiem